editor/find_in_files.cpp

```diff
@@ -724,7 +726,7 @@ void FindInFilesPanel::_on_result_found(String fpath, int line_number, int begin
 
 	Color file_item_color = _results_display->get_theme_color(SNAME("font_color")) * Color(1, 1, 1, 0.67);
 	file_item->set_custom_color(0, file_item_color);
-	file_item->set_selectable(0, false);
+	file_item->set_selectable(0, true);
```

Comment:
This makes the item not collapse when the text is clicked, making folding more difficult. Was it required to use the button?  I think not.

Reply:
Problem with this is that it makes it impossible to remove a file without removing all it's matches.

Reply:
Nvm just the wrong column.

Reply:
I opened #83065
You can now revert this line.